server/db/models/Order.js

```diff
@@ -27,45 +20,45 @@ Order.findAllWithProducts = async () => {
       model: OrderItem,
       attributes: ORDER_ITEM_PUBLIC_FIELDS,
       include: {
-        model: Product, 
-        attributes: PRODUCT_PUBLIC_FIELDS
+        model: Product,
+        attributes: PRODUCT_PUBLIC_FIELDS,
```

Comment:
seems like any reference to *_PUBLIC_FIELDS would throw an error - we may want to take these methods out this/next push